universum/modules/vcs/vcs.py

```diff
@@ -172,10 +172,12 @@ def clean_sources_silently(self):
         except OSError:
             pass
 
-    def supports_copy_cl_files_and_revert(self):
-        return self.driver.supports_copy_cl_files_and_revert
+    def copy_cl_files_and_revert(self) -> None:
```

Comment:
I don't think this function is needed here.

Reply:
removed